### DISCUSSION TITLE
Apply releases one by one - one at a time

### DISCUSSION THREAD

**Discussion by thevops** [helmfile/helmfile#955] [Questions] [answered]
Hey,

Is there any option to apply multiple releases one by one?
I'm using filtering and as a result I have e.g. 10 releases. I'd like apply them all, but one after another. Not in parallel.
It'd be even better to have the ability to add a delay between them. It can be done by adding a hook - but it will work only if there will be sequence apply, not parallel.

**Comment by yxxhero** [comment, marked as answer]
--concurrency int @thevops bro. here you are.

**Comment by thevops** [comment]
Wow! Thanks, this is it!

**Comment by thevops** [comment]
Is it possible to set `--concurrency` in helmfile.yaml ?

**Comment by yxxhero** [comment]
no for now. @thevops bro.
